validator/sawtooth_validator/execution/processor_handlers.py

```diff
@@ -26,6 +26,11 @@
 LOGGER = logging.getLogger(__name__)
 
 DEFAULT_MAX_OCCUPANCY = 10
+# This is the version used by SDK to match if validator supports feature it
+# requested during registration. It should only be incremented when there are
+# changes in TpRegisterRequest. Remember to sync this information in SDK if
+# changed.
+MAX_SDK_PROTOCOL_VERSION = 1
```

Comment:
Please break out the protocol version pieces into it's own commit. This may help w/backporting this piece to 1-1.

Reply:
Sure, there will be 2 commits. One introduction of protocol _version. Then introduction of version 1.